op-node/rollup/derive/engine_consolidate.go

```diff
@@ -26,6 +28,27 @@ func AttributesMatchBlock(attrs *eth.PayloadAttributes, parentHash common.Hash,
 	}
 	for i, otx := range attrs.Transactions {
 		if expect := block.Transactions[i]; !bytes.Equal(otx, expect) {
+			if i == 0 {
```

Comment:
This is a pretty unwieldy piece of code, what do you think about putting it into a function so its not all in the hot code path?

Reply:
done